README.rst

```diff
@@ -36,16 +36,14 @@ The required dependencies to use the software are:
 
 * Python >= 2.6,
 * setuptools
-* Numpy >= 1.6
+* Numpy >= 1.6.1
 * SciPy >= 0.9
 * Scikit-learn >= 0.12.1
-* Nibabel >= 1.1.0.
+* Nibabel >= 1.3.0
```

Comment:
> What about finding some middle ground and going for 1.2.2, which is the
> version the version of python-nibabel in Ubuntu 14.04?

We certainly should not go above that.

But what is the argument in favor of bumping the version? What is the
logic? What does it gain us and the user?